python/ray/train/tests/test_tensorflow_checkpoint.py

```diff
@@ -0,0 +1,67 @@
+import tensorflow as tf
+
+import ray
+from ray.train.batch_predictor import BatchPredictor
+from ray.train.tensorflow import (
+    TensorflowCheckpoint,
+    TensorflowTrainer,
+    TensorflowPredictor,
+)
+from ray.air import session
+from ray.air.config import ScalingConfig
+
+
+def test_tensorflow_checkpoint_saved_model():
```

Comment:
in addition to these integration tests can we also add some unit tests for TensorflowCheckpoint? Just like for each from_* format that `get_model` is working correctly.